tests/codex/testerasure.nim

```diff
@@ -16,6 +16,7 @@ import ./helpers
 
 suite "Erasure encode/decode":
 
+  const BlockSize = 64
```

Comment:
This is way too small to do anything meaningful, at least 1KB should be used.

Reply:
Thanks! I updated it to `1024`.